src/screens/Reports/Report.tsx

```diff
@@ -42,42 +44,43 @@ interface ListPropsInfo {
 }
 
 function Report(props: ReportProps) {
+  const intl = useInternationalization();
   const { classes, match } = props;
   const report = props.location.state.report;
   const deadline = props.location.state.deadline;
   const currentSite = useSelector(selectCurrentSiteInformation);
   const profit: CardPropsInfo[] = [{ number: roundToString(report.totalProfit), label: 'Profit', unit: 'Ks' }];
 
   // Customers
-  const collected = `${roundToString(report.totalAmountCollected)} Ks`;
-  const billed = `${roundToString(report.totalAmountBilled)} Ks`;
-  const outstanding = `${roundToString(report.totalAmountBilled - report.totalAmountCollected)} Ks`;
+  const collected = `${roundToString(report.totalAmountCollected)} ${intl(words.ks)}`;
+  const billed = `${roundToString(report.totalAmountBilled)} ${intl(words.ks)}`;
+  const outstanding = `${roundToString(report.totalAmountBilled - report.totalAmountCollected)} ${intl(words.ks)}`;
   const paymentRate = report.totalCustomersBilled ? `${roundToString(report.totalCustomersPaid / report.totalCustomersBilled * 100)}%` : '--';
   const customers = [
-    { label: 'Collected', value: collected},
-    { label: 'Billed', value: billed},
-    { label: 'Outstanding', value: outstanding },
-    { label: 'Payment Rate', value: paymentRate },
+    { label: intl(words.collected), value: collected},
+    { label: intl(words.billed), value: billed},
+    { label: intl(words.outstanding), value: outstanding },
+    { label: intl(words.payment_rate), value: paymentRate },
   ];
 
   // Inventory
-  const approved = `${roundToString(report.inventoryAmountApproved)} Ks`;
-  const rejected = `${roundToString(report.inventoryAmountDenied)} Ks`;
+  const approved = `${roundToString(report.inventoryAmountApproved)} ${intl(words.ks)}`;
+  const rejected = `${roundToString(report.inventoryAmountDenied)} ${intl(words.ks)}`;
   const inventory = [
-    { label: 'Collected', value: collected},
-    { label: 'Billed', value: billed},
-    { label: 'Outstanding', value: outstanding },
-    { label: 'Payment Rate', value: paymentRate },
+    { label: intl(words.collected), value: collected},
+    { label: intl(words.billed), value: billed},
+    { label: intl(words.outstanding), value: outstanding },
+    { label: intl(words.payment_rate), value: paymentRate },
```

Comment:
Heads up, this is a bug that @cnnmon has a PR to fix https://github.com/calblueprint/meepanyar/pull/150 . Since yours is the bigger change, I'm down with you merging first and Tiffany can fix any conflicts and merge. I'll let her know!